app/src/main/java/com/a0xffffffff/dinesum/RequestState.java

```diff
@@ -1,4 +1,24 @@
 package com.a0xffffffff.dinesum;
 
-public class RequestState {
+abstract public class RequestState {
+
+    // not using enum because hard to use with Firebase
+    public static final String PENDING = "pending";
```

Comment:
We can have these constants inside a globally accessible class instead of having a separate object for each state, for example, RequestState.PENDING

Reply:
Yeah we can do that if we decide not to use separate RequestState subclasses. See above comment for explanation why we thought we needed RequestState subclasses.